transitions/index.js

```diff
@@ -21,6 +21,7 @@ var _ = require('underscore'),
  */
 var AVAILABLE_TRANSITIONS = [
     'accept_patient_reports',
+    'add_info_document',
```

Comment:
I don't think this transition should be optional. Although you can live without it for now I think we'll soon start depending on the doc, so let's just hardcode it for everyone.